exampleCourse/questions/demo/studentFaces/info.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
There are still references to this question in `docs/howToLearnNames.md` and `docs/elements.md` - let's remove them if we're getting rid of this question too. Ditto for `demo/studentNames`.

Reply:
Removed in 7b7d1de